src/OrbitClientModel/include/OrbitClientModel/CaptureData.h

```diff
@@ -198,6 +198,10 @@ class CaptureData {
 
   [[nodiscard]] const ProcessData* process() const { return &process_; }
 
+  [[nodiscard]] bool has_post_processed_sampling_data() const {
```

Comment:
Aaand there you can see why I don't like these snake_case exceptions. This already doesn't fit in the snake_case cases that the style guide allows but at the same time it would be very weird to call it `HasPostProcessedSamplingData`. Just a little rant, ignore it.

Reply:
Well thats still the official snake_case google style.

Reply:
I mean it is still a very basic accessor.

Reply:
Well, kind of, it explicitly says "get and set functions". It's fine, protos also use `has_`. But when you add the fact that the two styles are mixed and the fact that it's not super clear what functions are included, it would just be cleaner to have a single style.

Reply:
That's said. In general I agree, and I don't like the style guides "use snake case" suggestion. However, we started with this, so I want to be strict with that, as otherwise, it will be even more mixture.